docs/install/kubernetes.md

```diff
@@ -29,7 +29,7 @@ For local installation on Windows or Mac, [Docker Desktop](https://www.docker.co
 - The Kubernetes command-line tool (kubectl) compatible with your Kubernetes cluster is installed. Refer to the [kubectl installation 
 instructions](https://kubernetes.io/docs/tasks/tools/install-kubectl/) for more information.
 - Helm >=3.6 CLI is installed. Refer to the [Helm installation instructions](https://helm.sh/docs/intro/install/) for more information.
-- An accessible docker-registry (such as [Docker Hub](https://hub.docker.com)). The registry's URL and credentials are consumed by the applications via a pre-created secret.
+- An accessible docker-registry (such as [Docker Hub](https://hub.docker.com)). The registry's URL and credentials are consumed by the applications via a pre-created secret. See the [Docker ID documentation](https://docs.docker.com/docker-id/) for details about creating a user with login that you will configure in the secret.
```

Comment:
missing the information about the default Docker Hub server - `If using Docker Hub, the registry server is `https://registry.hub.docker.com/``, I think it's worth adding this line because this line was deleted below

Reply:
added